packages/itwin/grouping-mapping-widget/src/widget/components/Grouping.tsx

```diff
@@ -548,67 +558,83 @@ export const Groupings = ({ mapping, goBack }: GroupsTreeProps) => {
+                                    close();
                                   }}
-                                />
-                                Edit
-                              </MenuItem>,
-                              <MenuItem
-                                key={1}
-                                onClick={async () => openProperties(g)}
-                                icon={<SvgList />}
-                              >
-                                Properties
-                              </MenuItem>,
-                              <MenuItem
-                                key={2}
-                                onClick={() => {
-                                  setSelectedGroup(g);
-                                  setShowDeleteModal(true);
-                                  close();
-                                }}
-                                icon={<SvgDelete />}
-                              >
-                                Remove
-                              </MenuItem>,
-                            ]}
+                                  icon={<SvgDelete />}
+                                  data-testid="gmw-context-menu-item"
+                                >
+                                  Remove
+                                </MenuItem>,
+                              ].concat(
+                                contextUIs.map((p) => {
+                                  return <MenuItem
+                                    key={p.name}
+                                    onClick={() => {
+                                      if(p.uiComponent) {
+                                        setSelectedGroup(g);
+                                        setSelectedContextCustomUI(p);
+                                        setGroupsView(GroupsView.CUSTOM);
+                                      }
+                                      if(p.onClick) {
+                                        p.onClick(g.id, mapping.id, iModelId);
+                                      }
+                                      close();
+                                    }}
+                                    icon={p.icon}
+                                  >
+                                    {p.displayLabel}
+                                  </MenuItem>;
+                                })
+                              )
+                            }
                           >
                             <IconButton
                               disabled={isLoadingQuery}
                               styleType='borderless'
+                              data-testid="gmw-more-button"
                             >
-                              <SvgMore
-                                style={{
-                                  width: "16px",
-                                  height: "16px",
-                                }}
-                              />
+                              <SvgMore />
```

Comment:
Why was the style override removed? Does it no longer require it? How did this change visually?

Reply:
Yeah the style is defined in the IconButton. The visual won't be affected and it's more consistent